### DISCUSSION TITLE
Addon unvailable

### DISCUSSION THREAD

**Discussion by Laurent-3D** [nutti/Screencast-Keys#147] [Q&A] [answered]
Blender 3.4.1 on os 12.6.2
I install via blender : No screencast add on
I install in manual mode : No screencast add on

What is the issue ?
ADDon preferences
<img width="654" alt="2023-01-17_10-57-01" src="https://user-images.githubusercontent.com/122872577/212867434-2222e9eb-4ce1-4a9f-bd0e-6214df7b2cb6.png">
System
<img width="1296" alt="2023-01-17_10-59-00" src="https://user-images.githubusercontent.com/122872577/212867935-d7829cde-59c0-4cb5-a17f-12ebe3aa175a.png">

Thank you

**Comment by nutti** [comment, marked as answer]
@Laurent-3D 

It seems that you mistake the installation process.

Please check the installation doc.
https://github.com/nutti/Screencast-Keys/blob/master/docs/installation.md

**Comment by Laurent-3D** [comment]
Yes I put new and its OK :-)

Thanks Nutti
